misc/k8s/templates/queue-deployment.js

```diff
@@ -0,0 +1,40 @@
+module.exports = {
+  apiVersion: 'extensions/v1beta1',
+  kind: 'Deployment',
+  metadata: {
+    creationTimestamp: null,
+    labels: { 'entropic-service': 'queue' },
+    name: 'queue'
+  },
+  spec: {
+    replicas: 1,
+    strategy: { type: 'Recreate' },
+    template: {
+      metadata: {
+        creationTimestamp: null,
+        labels: { 'entropic-service': 'queue' }
+      },
+      spec: {
+        containers: [
+          {
+            image: 'schickling/beanstalkd',
+            name: 'queue',
+            ports: [{ containerPort: 11300 }],
+            resources: {},
+            volumeMounts: [
+              { mountPath: '/var/lib/beanstalkd/', name: 'beanstalk-wal' }
```

Comment:
Wondering how it works with scaling. Didn't have experience with mounted volumes in k8s. 
Btw, curious to hear why for Postgres was chosen different path.

Reply:
Right now this one actually doesn't do anything, there's no code there yet: https://github.com/entropic-dev/entropic/tree/master/services/workers. It might have to be changed in the future. I've tried pretty hard to avoid volumes in k8s in general, which is why I didn't even include a Postgres deployment. Running containers in databases with persistent volumes just seems like more pain than it's worth (at least to me)

Reply:
I see. Thanks for the details. Yeah, persistent storage seems still to be terra incognita in k8s :)